### ISSUE TITLE
HTTPS authentication deprecated

### ISSUE THREAD

**Issue by ktyle** [ai2es/ai2es-template#2]
Since GitHub no longer allows https based user authentication for pushes/pulls when using a git command-line client (it is still possible, but requires you to set up a personal access token on GitHub), we should rewrite the 3rd step in the "Getting Started" section so that folks just use SSH. The[ Software Carpentry Git lessons](https://swcarpentry.github.io/git-novice/07-github/index.html) now recommend that. I'll put together a PR to address this.

**Comment by antoniomedrano** [comment]
Does this deprecate any of the procedures in githubcommand.md?

**Comment by ktyle** [comment]
@antoniomedrano I don't think so ... but (and this is a separate issue) we should probably standardize git-based workflows so the **master** branch is replaced with **main**, since that is the default now in GitHub (although, annoyingly, not with git yet) .

**Comment by antoniomedrano** [comment]
@ktyle Understandable the change, and it appears that it's pretty simple to rename branches. It's just a matter of reviewing all documentation and updating.

**Comment by ktyle** [comment]
Yes, one can also (with a version of git >=2.28) configure your local git environment so the default branch is always **main**: 

```
git config --global init.defaultBranch main

**Comment by antoniomedrano** [comment]
@ktyle I was running git version 2.21.0 on my mac but I got it up to 2.33.0 now. I'll confirm with you in the meeting before I run the config command. Thanks for the heads up!
